lib/kaffe/group_member/worker/worker_manager.ex

```diff
@@ -0,0 +1,63 @@
+defmodule Kaffe.WorkerManager do
+  @moduledoc """
+  This is the main process for launching group members and workers.
+
+  The process begins by starting the client connection to Kafka. Then, group
+  members are created for each of the configured topics.
+
+  This is also the place where subscribers get the PID for a Worker.
+  """
+  
+  use GenServer
+
+  alias Kaffe.WorkerSupervisor
+
+  require Logger
+
+  def start_link do
+    Logger.debug "Starting worker manager"
+    GenServer.start_link(__MODULE__, [self()])  
+  end
+
+  def worker_for(pid, partition) do
+    GenServer.call(pid, {:worker_for, partition})
+  end
+
+  ## Callbacks
+
+  def init([supervisor_pid]) do
+    Logger.info "event#starting=#{inspect self()}"
+
+    worker_table = :ets.new(:workers, [:set, :protected])
```

Comment:
Ahh I think I see. Could you update the moduledoc to make it clear this module is concerned with tracking partitions to workers? From the doc it looked like it was essentially recreating supervisor level knowledge of workers.